tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "moduleResolution": "nodenext",
     "resolveJsonModule": true,
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "plugins": [{ "name": "typescript-plugin-css-modules" }]
```

Comment:
TS plugins are only run on watch or by the IDE, they don't run on `build`.

https://www.npmjs.com/package/typescript-plugin-css-modules

The changes suggested by parcel "break" the `styles` import type (it's `any`) but this plugin restores full support (plus parcel also natively type-checks the specific imports even on `build`)